docs/EDID to Receiver Caps Mapping.md

```diff
@@ -61,11 +61,13 @@ Standard Timing Definitions (STD) format is defined in [E-EDID][E-EDID] section
 - `urn:x-nmos:cap:format:grain_rate` MUST be calculated with _Field Refresh Rate_
 - `urn:x-nmos:cap:format:interlace_mode` MUST be set to `progressive`
 
+Some of STDs are described in [DMT][DMT] with more precise vertical frequencies than _Field Refresh Rate_. Such STDs SHOULD be described with `urn:x-nmos:cap:format:grain_rate` supporting both frame rate values.
```

Comment:
Does DMT define just one additional (more precise) rate? If I've understood that correctly maybe rather than "both frame rate values", maybe "both the STD and DMT frame rate values" or "both the imprecise/standard and precise frame rate values" or something similar?


```suggestion
Some STDs are described in [Display Monitor Timing (DMT)][DMT] with more precise vertical frequencies than _Field Refresh Rate_. Such STDs SHOULD be described with `urn:x-nmos:cap:format:grain_rate` supporting both the STD and DMT frame rate values.
```

Reply:
Yes, it's a good candidate in the suggestion.

DMT describes the most popular video modes in detail from the waveform perspective, STD rather describes video modes how they're named. A video mode can be named `640x480@85Hz` but its precise vertical frequency per DMT is 85.008 Hz.

I don't think the DMT frame rate is what we expect to see in a Flow so I can't say it's better. `the STD and DMT` sounds good.

Reply:
If one STD rate always corresponds to the same precise DMT rate, i.e. 85 Hz is _always_ just a short hand convention for 85.008... Hz, and not sometimes 85.007 Hz, then what are the pros and cons of  IS-11 just insisting on using one of them?

Reply:
Sometimes the difference is bigger: e.g. `640x480@60Hz` has 59.94 Hz in DMT which seems a different video mode. E-EDID A2 section 3.9 says `The standard timing 2 byte codes for most VESA DMT definitions are listed in the latest revision of the DMT document`. That probably means that DMT keeps original information, and STD just refers to it which is a good reason for using DMT.

If nobody knows reasons for not doing it, I'll rewrite this paragraph.

Reply:
In this case is it that the correct grain rate is from the DMT? but a device is allowed to return only the nominal STD grain rate if it is not sensitive to the exact timing.

Reply:
My understanding is that when we describe a video mode via a Flow, it's up to the Device what are technical parameters to achieve the declared `grain_rate` and how constant it is. EDID works with timings which not always correspond to well-known video modes.

Standard Timings section describe only well-known modes as I can understand from its name, so that the video mode name is enough and every manufacturer should know what the timing for this mode is. Some of the timings are described in DMT.

It depends on the grain rate that baseband-IP gateways use for these well-known modes and timings and I don't know what this grain rate is. I guess, that the video modes described in DMT are common not only for the physical world, but also for the networked world but the corresponding timings are common only for the physical world and can't be exactly followed in the networked world. That's why I would choose the shorthands over the exact timings.

We can also mention that it's implementation-specific and give a vendor a choice between these two grain rates.

Reply:
E-EDID A2 section 3.8 says `Established Timings I & II indicate Factory Supported Modes of VESA DMT as well as other industry de-facto timings that predate EDID` so we have the same situation with Established Timings.